core/lib/instrument_api.h

```diff
@@ -6548,14 +6541,11 @@ DR_API
 /** Checks to see that if there is a trace in the code cache at tag \p tag. */
 bool
 dr_trace_exists_at(void *drcontext, void *tag);
-#    endif /* CUSTOM_TRACES */
-
-#endif /* CLIENT_INTERFACE */
+#endif /* CUSTOM_TRACES */
 
 /****************************************************************************
  * proc.c routines exported here due to proc.h being in arch_exports.h
  * which is included in places where opnd_t isn't a complete type.
- * These are used for dr_insert_clean_call() and thus are not just CLIENT_INTERFACE.
```

Comment:
Maybe keep this part: "These are used for dr_insert_clean_call()."

Reply:
Done.